doc/DESI_SPECTRO_REDUX/SPECPROD/zcatalog/ztile-SURVEY-PROGRAM-GROUPTYPE.rst

```diff
@@ -2,7 +2,10 @@
 ztile-SURVEY-PROGRAM-GROUPTYPE.fits
 ===================================
 
-:Summary: This file summarizes some set of files TBD.
+:Summary: This file contatenates the individual
+          :doc:`tile-based Redrock redshift catalogs </DESI_SPECTRO_REDUX/SPECPROD/tiles/GROUPTYPE/TILEID/GROUPID/redrock-SPECTROGRAPH-TILEID-GROUPID>`
+          into a single file per SURVEY, PROGRAM, and spectral GROUPTYPE.
+
```

Comment:
RST fussiness: I'm not sure how it renders with this blank line here.

Reply:
Between `:Summary:` and `:Naming Convention:` if that's not clear.

Reply:
Thanks.  It appears to render identically with or without the blank line, but the extra blank line was an unintended editing leftover so I removed it.